src/Illuminate/Support/Arr.php

```diff
@@ -120,7 +120,7 @@ public static function exists($array, $key)
             return $array->offsetExists($key);
         }
 
-        return array_key_exists($key, $array);
+        return isset($array[$key]) || array_key_exists($key, $array);
```

Comment:
Even then, you're making this twice as slow for cases where it is there and is null.